charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -68,59 +68,78 @@ data:
-              ephemeral-storage: "2Gi"
-            requests:
-              ephemeral-storage: "1Gi"
-
-          # Disable creation of initial user.
-          defaultUser:
-            enabled: false
-    {{- if .Values.global.azure.enabled }}
-          livenessProbe:
-            failureThreshold: 25
-            periodSeconds: 10
-    {{- end }}
-
-        # Worker configuration (applies to Celery and Kubernetes task pods).
-        workers:
-          resources:
-            limits:
-              ephemeral-storage: "2Gi"
-            requests:
-              ephemeral-storage: "1Gi"
-
-          # This is here for upgrading to 0.11.1+ of Airflow.
-          keda:
-            enabled: false
-            namespaceLabels:
-              name: {{ .Release.Name }}
-
-          # This is here for upgrading to 0.11.2+ of Airflow.
-          safeToEvict: true
-
-        # Scheduler configuration.
-        scheduler:
-
-          # Scheduler pod disruption budget
-          podDisruptionBudget:
-            enabled: false
-
-            # PDB configuration
-            config:
-              maxUnavailable: 1
-
-          # This is here for upgrading to 0.11.2+ of Airflow.
-          safeToEvict: true
+      {{- if .Values.global.sccEnabled }}
+        # If security context constraints are enabled, enable SCCs for airflow-chart
+        sccEnabled: true
+      {{- end }}
+        airflow:
+          useAstroSecurityManager: true
+          # Webserver configuration.
+          webserver:
+            extraVolumeMounts:
+              - name: signing-certificate
+                mountPath: /etc/airflow/tls
+                readOnly: true
+            extraVolumes:
+              - name: signing-certificate
+                secret:
+                  # This is the name of the secret that gets copied from the platform namespace
+                  # to the airflow namespaces.
+                  secretName: {{ template "houston.jwtCertificateSecret" . }}
+            resources:
+              limits:
+                # XXX: There is an alert configured to trigger when ephemeral storage reaches
+                # 1800000000 bytes (90% of the 2Gi limit). Be aware of this when modifying
+                # ephemeral-storage limits.
+                ephemeral-storage: "2Gi"
+              requests:
+                ephemeral-storage: "1Gi"
+
+            # Disable creation of initial user.
+            defaultUser:
+              enabled: false
+      {{- if .Values.global.azure.enabled }}
+            livenessProbe:
+              failureThreshold: 25
+              periodSeconds: 10
+      {{- end }}
+
+          # Worker configuration (applies to Celery and Kubernetes task pods).
+          workers:
+            resources:
+              limits:
+                ephemeral-storage: "2Gi"
+              requests:
+                ephemeral-storage: "1Gi"
+
+            # This is here for upgrading to 0.11.1+ of Airflow.
```

Comment:
Is this still needed? [0.11 is 7 years old](https://github.com/apache/airflow/tree/0.11)

Reply:
same as https://github.com/astronomer/astronomer/pull/1135/files#r712392264

Reply:
```suggestion
            # This is here for upgrading to 0.11.1+ of airflow-chart
```